src/browser/SplashScreenProxy.js

```diff
@@ -93,11 +94,13 @@ var SplashScreen = {
             localSplash = null;
             window.removeEventListener("resize", onResize, false);
 
+            var transitionCssString = "opacity " + fadeSplashScreenDuration + "ms ease-in-out";
+
             innerLocalSplash.style.opacity = '0';
-            innerLocalSplash.style["-webkit-transition"] = "opacity 1s ease-in-out";
-            innerLocalSplash.style["-moz-transition"] = "opacity 1s ease-in-out";
-            innerLocalSplash.style["-ms-transition"] = "opacity 1s ease-in-out";
-            innerLocalSplash.style["-o-transition"] = "opacity 1s ease-in-out";
+            innerLocalSplash.style["-webkit-transition"] =
+                innerLocalSplash.style["-moz-transition"] =
+                innerLocalSplash.style["-ms-transition"] =
+                innerLocalSplash.style["-o-transition"] = transitionCssString;
```

Comment:
Uh, does this work this way with `foo = bar = baz = value;`?

Reply:
Yes - wonders of JS :)

Reply:
The other thing I'd like to discuss: Do we even need the prefixes besides for the IE? Looking at caniuse (https://caniuse.com/#feat=css-transitions) the only browser which does not support it unprefixed is the IE <10

Reply:
I guess prefixes could be left out .. just didn't want to mess too much with the code.